### DISCUSSION TITLE
Frigate Plus Paid Subscription and API key used on two or more separate sites

### DISCUSSION THREAD

**Discussion by palinf** [blakeblackshear/frigate#14097] [Ask A Question]
I currently use the same Frigate Plus API key on two separate Frigate installations at different physical locations (two house I own), and it seems to be working without any issues. 
I'm planning to add a third installation soon. 
The documentation doesn't provide much information about using the same API key across multiple installations. Is there a limit on how many installations can be associated with a single key? Are there any potential drawbacks or considerations I should be aware of?

**Comment by Calimerorulez** [comment]
I would like to know this too...
